Calmspace/FourthQViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FourthQViewController.swift
+//  Calmspace
+//
+//  Created by Admin on 26/11/2023.
+//
+
+import UIKit
+
+class FourthQViewController: UIViewController {
```

Comment:
What's interesting, is that third and fourth view controllers are identical in structure: a question label, two buttons, and some background. The first couple are almost the same, they just have three buttons. 
A good exercise could be to create a single view controller with customizations:
1. question text
2. number of buttons
3. buttons titles and actions
4. background

and use it to display all the questions flow. What do you think?